### PR TITLE
Postman release notes

### PR DESCRIPTION
**Pull request by gcatanese**
Create Release Notes of the AdyenDev Postman resources

### CODE REVIEW

File: adyendev-postman-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# AdyenDev Release Notes
+
+The AdyenDev Postman space helps developers to explore, integrate and test the Adyen APIs.  
+
+This page contains the **Release Notes** of the following resources:
+* [Adyen APIs](#adyen-apis)
+* [Adyen in-person payments](#adyen-in-person-payments)
```
Comment (Kwok-he-Chu) [inline adyendev-postman-release-notes.md:7]: Capitalize `In-person Payments`

File: adyendev-postman-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- The path variables (i.e. `/action/{id}`) are no longer defined as environment variables. Edit them directly in the endpoint URL for more flexibility, enabling the testing of various scenarios and entities.
+
+### Feb 2023
+- First release: creation of the workspace and all Postman collections.
+
+---
+
+## Adyen in-person payments
+
+> Build a feature-rich point-of-sale integration with the Adyen in-person payments solution. See on [Postman].(https://www.postman.com/adyendev/workspace/adyen-in-person-payments/overview).
```
Comment (Kwok-he-Chu) [inline adyendev-postman-release-notes.md:28]: Link is not working, remove the `.` here

Comment (Kwok-he-Chu) [inline adyendev-postman-release-notes.md:28]: The ipp-team rebranded from Point-of-sale to In-person Payments, let's use In-person payments everywhere for consistency w/ docs

File: adyendev-postman-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+### Sept 2023
+- The path variables (i.e. `/action/{id}`) are no longer defined as environment variables. Edit them directly in the endpoint URL for more flexibility, enabling the testing of various scenarios and entities.
+
+### Feb 2023
+- First release: creation of the workspace and all Postman collections.
+
+---
+
+## Adyen in-person payments
```
Comment (Kwok-he-Chu) [inline adyendev-postman-release-notes.md:26]: Capitalize

**Review by Kwok-he-Chu — APPROVED** [review]

File: adyendev-postman-release-notes.md (hunk 1 above)
Comment (gcatanese) [reply adyendev-postman-release-notes.md:7]: I am using `in-person payments` as on docs https://docs.adyen.com/point-of-sale/

File: adyendev-postman-release-notes.md (hunk 3 above)
Comment (gcatanese) [reply adyendev-postman-release-notes.md:26]: Same here using `in-person payments` as on docs https://docs.adyen.com/point-of-sale/

**Pull request merged**